docs/docs/platform/metrics.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: 'Metrics (beta)'
+sidebar_position: 1
+image: /img/og/platform/metrics.png
+---
+
```

Comment:
maybe add a note about the feature being beta and being included in the pro plan at least during the beta